.github/workflows/qa.yml

```diff
@@ -0,0 +1,18 @@
+name: Check for Sign Off
+
+on:
+  workflow_call:
+    inputs:
```

Comment:
I think `inputs` here should actually be `secrets` to protect the token.

Reply:
That's a good point. I've made the update.